horizons/ai/aiplayer/productionbuilder.py

```diff
@@ -263,24 +263,24 @@ def display(self):
 			return
 
 		tile_colors = {
-			BUILDING_PURPOSE.ROAD:            ( 30,  30,  30),
```

Comment:
I found in fife.py this line (14918)
If it possible to make the same for hex, for example: `def addHexColored(self, *args):`?
```
    def addColored(self, *args):
        """
        addColored(InstanceRenderer self, Instance instance, int32_t r, int32_t g, int32_t b, int32_t a=128)
        addColored(InstanceRenderer self, Instance instance, int32_t r, int32_t g, int32_t b)
        """
        return _fife.InstanceRenderer_addColored(self, *args)
```